(AAA) Access for Enterprises guide (#36600)
DataDog/documentation: content/en/getting_started/access_for_enterprises/protecting_assets.md

## Common patterns

### Assets that should only be edited by the responsible team

This is the most common pattern. The assets themselves aren't sensitive (anyone can view them) but they are critical to a given team's workflows and should not be updated by others.

*For example, every team has a set of Dashboards and Monitors that reflect their on-call processes and SLOs. A developer from another team viewing the Dashboard is fine; accidentally editing it or deleting a Monitor is not.*

**Recommended approach:** Use [granular access controls][1] to restrict **Edit** of each asset to the owning Team, while leaving **View** open to the organization. These can be set per asset with the UI, API, or Terraform. Even when an entire organization can view an asset, [Team Tags][2] allow for personalized filtering of more relevant assets across Datadog using the Team filter.

This is also the right approach for the "create but not edit" pattern: users can freely create their own Monitors and Dashboards, but granular access controls prevent them from editing assets owned by other teams. The boundary is ownership, not a role-level permission.

> Certain Datadog assets offer access levels beyond Edit, View, and No Access. For example, Logs Pipelines separate who can manage an entire pipeline vs. who can edit the processors within it. Review the [granular access controls documentation][1] for asset-specific options.

Reviewer: ```suggestion
Certain Datadog assets offer access levels beyond Edit, View, and No Access. For example, Logs Pipelines separate who can manage an entire pipeline vs. who can edit the processors within it. For more details, see [granular access controls][1].
```

### Assets that should only be viewed by the responsible team

This is less common. The assets themselves contain sensitive information, and no one outside the designated group should see them at all. **Note:** View-level restrictions are supported for Notebooks and Dashboards.

*For example, a Notebook created in response to a particularly sensitive security incident should be completely hidden from anyone not involved in the response. Or an executive Dashboard with financial data may need to be restricted to specific leadership roles.*

**Recommended approach:** Use granular access controls to restrict **View** of that asset to the specific Team or set of users, with **No Access** granted to the rest of the organization.

## Admin override considerations

> As noted in [Assigning Users to Roles and Teams][3], restricting assets to specific teams affects everyone, including administrators. If your platform administrators are not members of the owning team, they are locked out.

Reviewer: If this is just 

```suggestion
Restricting assets to specific teams affects everyone, including administrators. If your platform administrators are not members of the owning team, they are locked out. For more information, see [Assigning Users to Roles and Teams][3].
```
